sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -55,6 +58,18 @@ class PolynomialTrendForecaster(BaseForecaster):
         If true, then include a feature in which all polynomial powers are
         zero. (i.e. a column of ones, acts as an intercept term in a linear
         model)
+    prediction_intervals : bool, default=False
+        Whether to compute prediction intervals.
+        If True, additional calculations are done during fit to enable prediction
+        intervals to be calculated during predict.
+        The prediction intervals are calculated according to
```

Comment:
Move this to a "references" section? Have a look at the `SARIMAX` docstring for an example.